### PR TITLE
Boards: Add Raspberry Pi 3

### PR DESCRIPTION
**Pull request by tombettany**
Add identifying features for the Raspberry Pi 3.

cc @radujipa @alex5imon @Ealdwulf

### CODE REVIEW

File: kano/utils/hardware.py [hunk 1]
```diff
@@ -177,6 +186,8 @@ def get_rpi_model(revision=None):
             model_name = RPI_2_B_KEY
         elif int(revision, 16) & 0x00FFFFFF == 0x00900092:
             model_name = RPI_ZERO_KEY
+        elif revision.strip() == 'a02082':
```
Comment (radujipa) [inline kano/utils/hardware.py]: You may want to be careful here because of the Overclocking flag. Try to use the same approach as for the other models.

Comment (tombettany) [inline kano/utils/hardware.py]: Thanks, I thought that it would be a shortcut but I guess not...

I'm fixing now

File: kano/utils/hardware.py [hunk 2]
```diff
@@ -90,6 +93,12 @@
         'cpu_profile': RPI_2_CPU_PROFILE,
         'performance': RPI_2_B_SCORE,
         'arch': "armv7"
+    },
+    RPI_3_KEY: {
+        'name': 'Raspberry Pi 3',
+        'cpu_profile': RPI_3_CPU_PROFILE,
+        'performance': RPI_3_SCORE,
+        'arch': "armv7"
```
Comment (Ealdwulf) [inline kano/utils/hardware.py]: Actually it's v8. Not that this will make a difference, as I carefully made chromium conditional on !=v6

**Comment by radujipa** [comment]
:shipit:

**Pull request merged**